python/torch_mlir_e2e_test/linalg_on_tensors_backends/refbackend.py

```diff
@@ -24,143 +24,90 @@
 
 
 def checkArgTypeIsSupported(ty):
-    SUPPORTED = [np.float32, np.float64, np.int32, np.int64, np.bool_]
-    assert ty in SUPPORTED, f"Only numpy arrays with dtypes in {SUPPORTED} are supported"
+  SUPPORTED = [np.float32, np.float64, np.int32, np.int64, np.bool_]
+  assert ty in SUPPORTED, f"Only numpy arrays with dtypes in {SUPPORTED} are supported"
+
+
+memref_type_to_np = {
+    "mrf32": np.float32,
+    "mrf64": np.float64,
+    "mri1": np.bool_,
+    "mri32": np.int32,
+    "mri64": np.int64
+}
+elemental_type_to_ctype = {
+    "i1": ctypes.c_bool,
+    "i64": ctypes.c_int,
+    "f32": ctypes.c_float,
+    "f64": ctypes.c_double
+}
+
+
+def get_return_funcs(module):
+  refbackend_return_initials = "refbackend_consume_func_return_"
+  return_funcs = []
+  with Context() as ctx:
+    for func in module.body:
+      func_name = str(func.attributes["sym_name"])
+      if func_name[1:len(refbackend_return_initials) +
+                   1] == refbackend_return_initials:
+        return_funcs.append(func_name.replace('"', ''))
+
+  return return_funcs
+
+
+def get_ctype_func(func_name):
+  ret_types = func_name.split("_")[4:]
+  ctypes_arg = [None]
+  for type in ret_types:
+    if type in elemental_type_to_ctype:
+      ctypes_arg.append(elemental_type_to_ctype[type])
+    elif type in memref_type_to_np:
+      ctypes_arg.append(ctypes.POINTER(UnrankedMemRefDescriptor))
+    else:
+      assert True, f"Not supported type: {type}"
```

Comment:
> I think the indentations are off. Should be 4 spaces instead of 2 spaces.

I used this style https://github.com/llvm/torch-mlir/blob/main/.style.yapf (the indent width is mentioned to be 2)

Reply:
I have updated the indentation width and set it to 4.

Reply:
I just updated the style file to use pep8 style https://github.com/llvm/torch-mlir/blob/main/.style.yapf